.github/workflows/bump-cli.yml

```
@@ -1,6 +1,19 @@
 name: Bump CLI version
 on:
   workflow_dispatch:
+    inputs:
+      option:
+        description: "Option"
+        required: true
+        default: 'replace'
+        type: choice
+        options:
+          - prepend
+          - replace
+      version:
+        description: "Version"
```

I think more description is required here. How about:

```suggestion
        description: |
            The version to prepend to the supported versions file. This should be in the form: `vA.B.C`.
```